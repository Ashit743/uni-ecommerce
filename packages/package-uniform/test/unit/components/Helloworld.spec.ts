import  HelloWord  from "../../../src/components/HelloWorld.vue"
import { describe,it,expect } from 'vitest';
import { mount } from '@vue/test-utils'

describe('Hello world',()=>{
    let wrapper = mount(HelloWord)
    it('first sample test',()=>{
        expect(wrapper.element).toMatchSnapshot();
    })
})