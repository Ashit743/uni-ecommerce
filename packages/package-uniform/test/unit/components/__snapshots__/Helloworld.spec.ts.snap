// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Hello world > first sample test 1`] = `
<div
  data-v-app=""
>
  
  <h1
    data-v-e17ea971=""
  />
  <div
    class="card"
    data-v-e17ea971=""
  >
    <button
      data-v-e17ea971=""
      type="button"
    >
      count is 0
    </button>
    <p
      data-v-e17ea971=""
    >
       Edit 
      <code
        data-v-e17ea971=""
      >
        components/HelloWorld.vue
      </code>
       to test HMR 
    </p>
  </div>
  <p
    data-v-e17ea971=""
  >
     Check out 
    <a
      data-v-e17ea971=""
      href="https://vuejs.org/guide/quick-start.html#local"
      target="_blank"
    >
      create-vue
    </a>
    , the official Vue + Vite starter 
  </p>
  <p
    data-v-e17ea971=""
  >
     Learn more about IDE Support for Vue in the 
    <a
      data-v-e17ea971=""
      href="https://vuejs.org/guide/scaling-up/tooling.html#ide-support"
      target="_blank"
    >
      Vue Docs Scaling up Guide
    </a>
    . 
  </p>
  <p
    class="read-the-docs"
    data-v-e17ea971=""
  >
    Click on the Vite and Vue logos to learn more
  </p>
  
</div>
`;
